XamarinCommunityToolkit.UnitTests/Converters/StringToDoubleConverter_tests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Globalization;
+using Xamarin.CommunityToolkit.Converters;
+using Xunit;
+
+namespace Xamarin.CommunityToolkit.UnitTests.Converters
+{
+	public class StringToDoubleConverter_tests
+	{
+		[Theory]
+		[InlineData("2.2", 2.2)]
+		[InlineData("3.9", 3.9)]
+		[InlineData("255.98", 255.98)]
+		[InlineData("99191.98765", 99191.98765)]
```

Comment:
Can we also maybe add some test cases here that we expect to fail?

Reply:
BTW, "2.2" will probably fail for the German locale as well as "2,2" will fail for the US locale.

Reply:
@AndreiMisiukevich then how are doubles defined  if the . is equivalent to the , ?

Reply:
What do you mean?

Reply:
"BTW, "2.2" will probably fail for the German locale as well as "2,2" will fail for the US locale." i didn't understand this

Reply:
decimal separator depends on the locale

https://stackoverflow.com/questions/2583362/how-to-convert-string-to-double-with-proper-cultureinfo

So if somebody from Germany will try to run unit tests, they probably will fail (because in German locale a comma is used as a decimal separator)